### ISSUE TITLE
Inquiry: Discrepancy in Reproduced Results

### ISSUE THREAD

**Issue by hunanghuan03** [qiyan98/TrajFlow#4]
Hello Authors,

First of all, thank you for your excellent work and for making this repository publicly available. I'm currently working on reproducing your results and have a question about the outcome.

I've followed the documentation closely, but I'm running on a slightly different hardware and software configuration:

    CUDA Version: 12.1

    GPU: 2x NVIDIA RTX 4090 (running in parallel)

    Batch Size: Adjusted to 8

Under this setup, the process completes successfully, but I'm observing a significant difference between my results and the ones you have published.

I was wondering if this level of discrepancy is expected given the changes in the environment (GPU architecture, CUDA version) and hyperparameters (batch size). If this is indeed unusual, could you possibly shed some light on what might be causing such a large gap? I'm trying to determine if it stems from the hardware differences or if there are other factors I might need to investigate.

Any insights you could provide would be immensely helpful. Thank you for your time and for this fantastic project.

Best regards,

Your Huan

**Comment by hunanghuan03** [comment]
<img width="556" height="111" alt="Image" src="https://github.com/user-attachments/assets/004738ab-bec1-4f47-94ae-470fe27d6990" />

**Comment by qiyan98** [comment]
Thank you for your interest in our work. From your screenshot, it looks like these might be the validation set results. We actually re-ran the experiments on 4×V100 32GB GPUs after releasing the code and were able to obtain results very close to those reported in the paper.

On the full validation set, here is a brief summary of the performance from the last few checkpoints. These results were evaluated against the offline validation set (standard motion prediction track).

<img width="2533" height="1347" alt="Image" src="https://github.com/user-attachments/assets/c6ef07c3-1d6e-4f6a-bd0b-a278c21201f0" />

Without applying any careful checkpoint selection, we obtained the following results on the motion prediction test set:

<img width="1030" height="746" alt="Image" src="https://github.com/user-attachments/assets/72f6b285-64b1-4559-aa3e-b4e066f177c2" />

From my past experience, an average minADE larger than 0.75 usually suggests that the model has not been fully trained yet. Below is a curve of our online training minADE evaluation results (computed on a 10% standard validation split to speed up inference). While approximate, this still provides a reasonable indication of how the performance improves as training progresses.

<img width="423" height="262" alt="Image" src="https://github.com/user-attachments/assets/ba8ed94c-d808-443b-aebf-fc7b11121186" />

**Comment by hunanghuan03** [comment]
Thank you so much for this incredibly detailed and helpful response. I truly appreciate you taking the time to share not only the benchmark results but also your personal experience and the reference training curve.

The diagnostic tip that an average minADE > 0.75 likely indicates incomplete training is a particularly valuable insight. This gives me a very clear direction for my next steps.

Based on your advice, I will run more experiments, focusing on extending the training duration to ensure the model converges properly.
